schemas/treatment.json

```diff
@@ -112,6 +112,15 @@
         "displayName": "Is Primary Treatment"
       }
     },
+    {
+      "name": "line_of_treatment",
+      "description": "Indicate the line of treatment if it is not the primary treatment.",
+      "valueType": "integer",
+      "meta": {
+        "displayName": "Line of treatment",
```

Comment:
Line of `T`reatment

 i think all Display names have been using Title case minus (of/the/in)

Reply:
I will correct the casing so it's consistent.